distribution/src/resources/config-tool/default.json

```diff
@@ -1,58 +1,58 @@
 {
   "transport.passthru_http.listener.class": "org.apache.synapse.transport.passthru.PassThroughHttpListener",
-  "transport.passthru_http.parameters": {
+  "transport.passthru_http.parameter": {
     "port": "8280",
     "non-blocking": true,
     "httpGetProcessor": "org.wso2.carbon.mediation.transport.handlers.PassThroughNHttpGetProcessor"
   },
   "transport.passthru_https.listener.class": "org.apache.synapse.transport.passthru.PassThroughHttpSSLListener",
-  "transport.passthru_https.listener.parameters": {
+  "transport.passthru_https.listener.parameter": {
     "port": "8243",
     "non-blocking": true,
     "HttpsProtocols": "TLSv1,TLSv1.1,TLSv1.2",
     "httpGetProcessor": "org.wso2.carbon.mediation.transport.handlers.PassThroughNHttpGetProcessor"
   },
-  "transport.http.listener.key_store.file_name": "repository/resources/security/$conf{keystore.tls.file_name}",
-  "transport.http.listener.key_store.type": "$conf{keystore.tls.type}",
-  "transport.http.listener.key_store.password": "$conf{keystore.tls.password}",
-  "transport.http.listener.key_store.key_password": "$conf{keystore.tls.key_password}",
-  "transport.http.listener.trust_store.file_name": "repository/resources/security/$conf{truststore.file_name}",
-  "transport.http.listener.trust_store.type": "$conf{truststore.type}",
-  "transport.http.listener.trust_store.password": "$conf{truststore.password}",
+  "transport.http.listener.key_store.file_name": "repository/resources/security/$ref{keystore.tls.file_name}",
+  "transport.http.listener.key_store.type": "$ref{keystore.tls.type}",
```

Comment:
Shall we use single form for keystore? Either key_store or keystore. Same with truststore